quadrat/sass/elements/_forms.scss

```diff
@@ -0,0 +1,21 @@
+input[type="text"],
+input[type="email"],
+input[type="url"],
+input[type="password"],
+input[type="search"],
+input[type="number"],
+input[type="tel"],
+input[type="range"],
+input[type="date"],
+input[type="month"],
+input[type="week"],
+input[type="time"],
+input[type="submit"],
+input[type="datetime"],
+input[type="datetime-local"],
+input[type="color"],
+textarea {
+	&:focus {
+		outline: 1px dotted currentColor;
```

Comment:
This felt very theme specific Maybe it should move to BCB, but I feel like it will be awhile (if ever) before Gutenberg / theme.json provides this kind of control to the themers let alone users.

Reply:
Yea the "make it easy to do again" part of my brain agrees it should be in BCB.

The "BCB is for ponyfilling what Gutenberg will do" part of my brain is winning though and I agree with you; this belongs here.

Reply:
I think if we end up doing this in a few themes we should move it to BCB.